Formula/percolator.rb

```diff
@@ -0,0 +1,35 @@
+class Percolator < Formula
+  # cite K_ll_2007: "https://doi.org/10.1038/nmeth1113"
+  desc "Semi-supervised learning for peptide identification from shotgun proteomics data"
+  homepage "http://percolator.ms"
+  url "https://github.com/percolator/percolator/archive/rel-3-05.tar.gz"
+  version "3.05"
+  sha256 "5b746bdc0119a40f96bc090e02e27670f91eeb341736911750b170da7e5c06bb"
+  license all_of: ["Apache-2.0", "BSD-2-Clause", "BSD-3-Clause", "MIT"]
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "xerces-c" => :build
+  depends_on "xsd" => :build
+  depends_on "curl"
```

Comment:
Is `curl` really needed as a dependency? Does `brew linkage percolator` report run-time library linkage to `curl`?

Reply:
I believe so?

`brew linkage percolator` on my Ubuntu test environment, at `1d1ca6e`: (Admittedly this is including the `-lcurl` that I added to try and fix the CI curl failures)

<details>

```
brew linkage percolator
System libraries:
  /lib/x86_64-linux-gnu/libc.so.6
  /lib/x86_64-linux-gnu/libm.so.6
  /lib/x86_64-linux-gnu/libpthread.so.0
Homebrew libraries:
  /home/linuxbrew/.linuxbrew/lib/libcurl.so.4 (curl)
  /home/linuxbrew/.linuxbrew/lib/libgcc_s.so.1 (gcc)
  /home/linuxbrew/.linuxbrew/lib/libgomp.so.1 (gcc)
  /home/linuxbrew/.linuxbrew/lib/libstdc++.so.6 (gcc)
```
</details>

Reply:
Okay! Thanks for checking.